frontend/src/components/MeetingRoom/AudioControlButton/AudioControlButton.tsx

```diff
@@ -33,7 +33,7 @@ const AudioControlButton = (): ReactElement => {
   };
 
   return (
-    <div className="hidden xs:inline">
```

Comment:
These buttons should not be hidden. It was causing issues below our supported minimum width, but still should be changed, imo.